Added API_KEY and device id to all examples.

Vinayakj009 opened the pull request:
Have added API_KEY and DEVICE_ID macro parameter to all examples, so that users can quickly change this parameter, and accordingly use it with their codes.

Commit:
Added API_KEY and device id to all examples.

Commit:
Made it so that API_KEY and DEVICE_ID does not redefined, in all examples.

Commit:
Add common file for storing api key and device id

Commit:
Added data to readme to explain how to define global API key and device id.

Commit:
Changed version number, and moved url for logged in docs to public docs.

Vinayakj009 committed:
Github only lets you download the library with a master keyword at the end.

sashoeb94 commented on README.md:
```diff
@@ -8,11 +8,25 @@ Please follow the steps to include the bolt library to arduino
+# (Optional) Defining global API key and device id (Details of why this is needed will be added later)
+While it is a good idea to define the API key and device id as a part of the code, it is sometimes troublesome to write the corresponding code everytime you have a new project.
+You can define a global API key or device id, by using the following steps.
+1. Go to the 'libraries/boltiot-arduino-helper-master/src' folder inside your sketchbook folder. The sketchbook folder is normally 'My Documents/Arduino' folder for windows and "~/Documents/Arudino/" folder in Mac and linux. 
```
@Vinayakj009 Typo in windows installation path "~/Documents/Arudino"

gopherengine commented:
I hav'nt got the Ok button after importing the lib.

Vinayakj009 committed:
Update README.md

Corrected typo

Vinayakj009 committed:
Update README.md

Corrected typo of typo

rsbuilds commented on examples/CommandHandler/CommandHandler.ino:
```diff
@@ -7,6 +7,12 @@
+#define API_KEY   "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+#endif
+#ifndef DEVICE_ID
+#define DEVICE_ID "BOLTxxxxxxxx"
```
@Vinayakj009  Can we make #define DEVICE_ID some sort of list? Where I can store the multiple devices.

Vinayakj009 replied:
While a list can be defined, having the OTA update recognize the list, is not a part of the repo.
Also, I am planning on the ability to define a product id, such that all devices linked to the product will get OTA update.

rsbuilds commented on library.properties:
```diff
@@ -1,10 +1,10 @@
 paragraph= This library has been developed to reduce the effort required to interface the Arduino with the Bolt IoT platform.
 category=Communication
-url=https://cloud.boltiot.com/documentation
+url=https://cloud.boltiot.com/docs
```
@Vinayakj009 Confirm with @pranavkundaikar if it should be docs.boltiot.com

Vinayakj009 replied:
I do not think I should change it here.
This link already re-directs to the new documentation.

Also, I feel we should make changes in the Bolt Cloud such that, all old links re-direct to the new documentation, rather then making changes in where we have placed the links.

Vinayakj009 committed:
Update README.md

1) Changed quoted text to underlined text
2) Added line to show how to include device credentials.
3) Change begin from capital B to small b

Vinayakj009 committed:
Update README.md

Added cpp language highlighting to example code.
